site/content/in-dev/unreleased/admin-tool.md

```diff
@@ -17,20 +17,13 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-linkTitle: Polaris Admin Tool
-title: Apache Polaris (incubating) Admin Tool
+linkTitle: Admin Tool
+title: Admin Tool
 type: docs
 weight: 300
 ---
 
-In order to help administrators manage their Polaris metastore, Polaris provides an administration
-tool.
-
-## How to Download the Admin Tool
```

Comment:
Remove it as there is no place to download it now. Also, this doc may focus on more usage instead of building